node/parallel/src/chain_spec/mod.rs

```diff
@@ -15,13 +15,26 @@
 pub mod heiko;
 pub mod parallel;
 
-use primitives::*;
 use sc_chain_spec::{ChainSpecExtension, ChainSpecGroup};
+use sc_service::Properties;
 use serde::{Deserialize, Serialize};
+use serde_json::json;
 use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 use sp_core::{sr25519, Pair, Public};
 use sp_runtime::traits::IdentifyAccount;
 
+use primitives::{network::NetworkType, *};
+
+pub(crate) fn as_properties(network: NetworkType) -> Properties {
+    json!({
+        "ss58Format": network.ss58_addr_format_id(),
+        "network": network,
```

Comment:
hi @alannotnerd  where will this `network` field be used?

Reply:
It effects the display information in polkadotjs dapps, just like `tokenSymbol` field.

Reply:
Sorry, I checked the polkadotjs source code just now and found it's not the standard field in properties. I was thounght it's the name displayed in upper left corner. But in fact, it's from `api.system.name()`. I'll remove it later. BTW, what are the token name and its decimal? I can add them in the next commit incidentally.

Reply:
> Sorry, I checked the polkadotjs source code just now and found it's not the standard field in properties. I was thounght it's the name displayed in upper left corner. But in fact, it's from `api.system.name()`. I'll remove it later. BTW, what are the token name and its decimal? I can add them in the next commit incidentally.

no problem, we have `HKO` for heiko network and `PARA` for parallel network, they all have 12 as token decimals